index.js

```diff
@@ -1,17 +1,44 @@
 export const getUserChoice = (userInput) => {
-  // TODO
+  userInput = userInput.toLowerCase().trim();
+
+  switch (userInput){
+    case 'rock': return 'rock';
+    case 'paper': return 'paper'; 
+    case 'scissors': return 'scissors';
+    case 'bobm': return 'bomb'; 
+    default: console.log(`Please choose one of the words : "rock", "paper", or "scissors"`); break;
+  }
+
 };
 
 export const getComputerChoice = () => {
-  // TODO
+
+  let randDigit = Math.floor(Math.random() * 3);
+  if (randDigit === 0) return 'rock';
+  if (randDigit === 1) return 'paper';
+  if (randDigit === 2) return 'scissors';
 };
 
 export const determineWinner = (userChoice, computerChoice) => {
-  // TODO
+  if (userChoice === computerChoice) return 'tie';
+  
+  switch (userChoice){
+    case 'bomb': return 'user won'; 
```

Comment:
Mana shu qator shartmas chunki kompyuter baribir ham `bomb` tanlamaydi va "user won" qaytaveradi.

Reply:
Ha, tushunarli

Reply:
changes added deb commit qilib to'g'irlab qo'ydim